src/components/phone/PhoneOverlay.vue

```diff
@@ -432,4 +575,23 @@ const {
   </div>
 </template>
 
-<style scoped></style>
+<style scoped>
+/* filter for hex #f9cd00   */
+.filter-yellow {
+  filter: brightness(0) saturate(100%) invert(76%) sepia(21%) saturate(2390%)
+    hue-rotate(5deg) brightness(103%) contrast(104%);
+}
+/* filter for hex #7e7e81   */
+
+.filter-gray {
+  filter: brightness(0) saturate(100%) invert(83%) sepia(0%) saturate(0%)
+    hue-rotate(232deg) brightness(90%) contrast(90%);
+}
```

Comment:
These filters are already defined in BaseIcon. We need to make them global
https://github.com/CrisisCleanup/crisiscleanup-4-web/blob/23111fb91ce386de5ddf79c224976a49b2dcba4c/src/components/BaseIcon.vue#L184-L191